src/v2/Apps/Partner/Components/NavigationTabs/index.tsx

```diff
@@ -8,6 +8,8 @@ import { Sticky } from "v2/Components/Sticky"
 import { ScrollIntoView } from "v2/Utils"
 import { Media } from "v2/Utils/Responsive"
 import { NavigationTabs_partner } from "v2/__generated__/NavigationTabs_partner.graphql"
+import { useSystemContext } from "v2/System"
+import { userIsAdmin } from "v2/Utils/user"
```

Comment:
In general, for internal testing, it's better to use [userIsTeam](https://github.com/artsy/force/blob/master/src/v2/Utils/user.ts#L42) as we are stricter now about who has the `admin` role!